cedar-policy-validator/src/type_error.rs

```diff
@@ -267,9 +277,12 @@ pub struct TypesMustMatch {
 
 /// Structure containing details about a missing attribute error.
 #[derive(Debug, Hash, Eq, PartialEq)]
-pub struct MissingAttribute {
-    missing: String,
+pub struct UnsafeAttributeAccess {
```

Comment:
Can we pass in this struct to `fn unsafe_attribute_access(` instead of passing the `attribute`, `suggestion` and `may_exist` arguments?

Reply:
That functions is supposed to be a constructor for this struct, so it wouldn't make sense IMO